docs/source/getting_started.rst

```diff
@@ -0,0 +1,21 @@
+Getting Started
```

Comment:
How about calling this "Introduction"? "Getting Started" typically indicates set-up instructions, but that's not really what this page offers.

Reply:
So this was taken from https://opensource.com/article/20/3/documentation. It should meet the objectives defined in the section "Start with Getting Started documentation". So I'll keep in mind that it's currently not doing that 😄